src/filters/filters-context.js

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+
+const FiltersContext = React.createContext();
+
+const setFiltersType = "SET_FILTERS";
+
+function filtersReducer(state, action) {
+  switch (action.type) {
+
+    case setFiltersType: {
+      return {
+        ...state,
+        filters: action.filters
+      }
+    }
+    default: {
+      throw new Error(`Unsupported action type: ${action.type}`);
```

Comment:
I don't know if this is a redux specific convention, but rather than throwing on the `default` we tend to return the existing state to keep the site working.

With scoped states like this, I get thats not necessary though..